src/client/protocolgamesend.cpp

```diff
@@ -1321,14 +1321,21 @@ void ProtocolGame::sendMarketLeave()
     send(msg);
 }
 
-void ProtocolGame::sendMarketBrowse(const uint8_t browseId, const uint16_t browseType)
-{
+void ProtocolGame::sendMarketBrowse(const uint8_t browseId, const uint16_t browseType, const uint8_t tier)
+{    
     const auto& msg = std::make_shared<OutputMessage>();
     msg->addU8(Proto::ClientMarketBrowse);
     if (g_game.getClientVersion() >= 1251) {
         msg->addU8(browseId);
         if (browseType > 0) {
             msg->addU16(browseType);
+            // If browseId is 3 (browse item), send tier if item has classification
+            if (browseId == 3) {
+                const auto& item = Item::create(browseType);
```

Comment:
```suggestion
                const auto& item = Item::create(browseType);
```
What is the point of **creating** an item if you are not going to use it? You will only obtain information from getClassification.

better to use getThingType, for information purposes only

Reply:
Fixed, thanks for the comment.